pom.xml

```diff
@@ -575,6 +575,7 @@
               <exclude>**/nexus-csrfguard/**/org/sonatype/nexus/csrfguard/CsrfGuardFilter.java</exclude>
               <exclude>**/nexus-csrfguard/**/org/sonatype/nexus/csrfguard/CsrfGuardServlet.java</exclude>
               <exclude>**/nexus-ldap-common/**/org/sonatype/security/ldap/LdapEncoder.java</exclude>
+              <exclude>**/nexus-core/**/com/bolyuba/nexus/plugin/npm/internal/Version.java</exclude>
```

Comment:
should be **/nexus-npm-repository-plugin/**/com/bolyuba/nexus/plugin/npm/internal/Version.java